### DISCUSSION TITLE
Error when using npm run dev to start Nuxt 3

### DISCUSSION THREAD

**Discussion by mike-420** [nuxt/nuxt#10856] [Questions (Nuxt 2)]
I'm getting the following error when I type **npm run dev** after installing Nuxt 3.

**✖ Nuxt Fatal Error                                                            │
   │                                                                                 │
   │   TypeError: Cannot destructure property 'nuxt' of 'this' as it is undefined.**

I used the following command line to install it. It seemed to install okay until I entered npm run dev.
yarn create nuxt-app

**Comment by ndragun92** [comment]
If u are using the old Node Version please check that and use something newer.
If Node Version is 14.X something then remove node_modules and package-lock and do fresh install and then try again
